index.js

```diff
@@ -42,31 +42,43 @@ function init({
     batch = chunk;
   };
 
-  const aggregator = async args => {
-    let { hits, cursor } = args;
-    do {
-      if (!hits) {
-        return;
-      }
-      batch = batch.concat(
-        hits.reduce((entries, hit) => {
+  const aggregator = (() => {
+    var _ref3 = _asyncToGenerator2(function* (args) {
```

Comment:
do you mind using the source `async` code instead of the style in the built form? I'll gladly accept and release then :)

Thanks!